tests/api/test_experiments.py

```
@@ -464,7 +460,9 @@ def test_zero_repetitions_edge_case(self, simple_problem):
         assert jnp.isnan(fd_results.gradient_estimates["fd"]["std"]).all()
 
         # Also test the internal function directly to ensure coverage
-        from mellowgate.api.experiments import _compute_sweep_statistics
+        from mellowgate.api.experiments import (
+            _compute_sweep_statistics,  # noqa: PLC2701
```

Ruff (0.14.1)

Unused `noqa` directive (non-enabled: `PLC2701`)

Remove unused `noqa` directive

(RUF100)